build/tutorials/tutorials-contest/red-dev-avalanche-chainlink-tutorial/README.md

```diff
@@ -0,0 +1,441 @@
+
+![chainlink-bridge](img/Chainlink-bridge.png)
+
+## Create a new job
+
+The next step is to create a new job in the Chainlink node. 
+
+### Using type: "web"
+Below is a code snippet for the job specification to test the job within the Chainlink GUI. This can be done by using the "type": "web". Please check the Chainlink official [documentation](https://docs.chain.link/docs/job-specifications/) for more details. Later, in this section we also cover the "type": "runlog" which will be used to integrate Chainlink with on-chain contracts. 
+
+```
+{
+    "name": "Avalanche nodejs adapter test",
+    "initiators": [
+        {
+            "type": "web"
+        }
+    ],
+    "tasks": [
+        {
+            "type": "avalanchenodejs",
+            "params": {
+                "chain": "P",
+                "method": "platform.getCurrentSupply",
+                "params": {}
+            }
+        }
+    ]
+}
+```
+Now, in the jobs section, the newly created _Avalanche nodejs adapter test_ will be listed. Select that and click the Run button to test and see the result.
+
+![chainlink-job-run](img/Chainlink-job-run.png)
+
+Then, click on the Runs tab, and select the job id to see the result. It should be in _Completed_ status:
+
+![chainlink-job-result](img/Chainlink-job-result.png)
+
+### Using type: "runlog"
+
+The [RunLog](https://docs.chain.link/docs/initiators/) initiator is the easiest initiator to use when integrating Chainlink with on-chain contracts.
+```
+{
+  "name": "Avalanche nodejs adapter",
+  "initiators": [
+    {
+      "type": "runlog",
+      "params": {
+        "address": "0xac830beb7a2f1ced128e347e6b9a37dcc2e971b7"
+      }
+    }
+  ],
+  "tasks": [
+    {
+      "type": "avalanchenodejs",
+      "params": {
+        "chain": "P",
+        "method": "platform.getCurrentSupply",
+        "params": {
+        }
+      }
+    },
+    {
+      "type": "jsonparse"
+    },
+    {
+      "type": "ethuint256"
+    },
+    {
+      "type": "ethtx"
+    }
+  ]
+}
+```
+
+* jsonparse, ethuint256 & ethtx are core adaptors and are executed synchronously. For more details, you will find documentation for each adapter's usage [here](https://docs.chain.link/docs/core-adapters/).
+* runlog - By adding the address parameter, you make the event filter of the RunLog initiator more restrictive, only listening for events from that address, instead of any address. By adding the requesters parameter, you only allow requests to come from an address within the array. Please have a look at this [video](https://www.youtube.com/watch?v=ZB3GLtQvgME&38m38s) to learn how to get your oracle address.
+
+
+# Smart contracts
+
+This section will explain the smart contracts we have written for communicating with the Chainlink node through oracle address and job id.
+
+Below is the code snippet which you can copy and paste into the Remix IDE and then compile and deploy.
+
+```
```

Comment:
Could you please add format definition to all code sections? Like:
\`\`\`solidity
soliditiy code goes here
\`\`\`
It helps with syntax highlighting

Reply:
Added as suggested. Please have a look at 4ba5be59729a9bc2d8bf6a840c768ba00355cd4b